output.c

```diff
@@ -132,6 +132,12 @@ typedef struct Output_t
 
 } Output_t;
 
+typedef struct ThreadData_t
```

Comment:
Please rename ThreadData_t to OutputThread_t

Reply:
Done.